mkosi/__init__.py

```diff
@@ -4126,14 +4127,18 @@ def secure_boot_sign(
         return
 
     with mount():
-        for path, _, filenames in os.walk(root / "efi"):
+        base_path = root / "efi" if esp else root / "usr/lib/systemd/boot/efi"
```

Comment:
Should we maybe always the stuff in `/usr/lib/systemd/boot/efi`? In that case it would maybe also make sense to lift which path is being searched for things to sign out of this function and make this path an argument (in lieu of `root`).

Reply:
Fixed.